test/runtests.jl

```diff
@@ -4,6 +4,7 @@ using Test
 @test_throws ArgumentError Blosc.set_num_threads(0)
 @test_throws ArgumentError Blosc.set_num_threads(Blosc.MAX_THREADS + 1)
 @test Blosc.set_num_threads(1) == 1
+@test Blosc.set_num_threads() == 1
```

Comment:
```suggestion
@test 1 <= Blosc.set_num_threads() <= Blosc.MAX_THREADS
```
Might be safer since CI could be configured to use more than 1 thread